devices.json

```diff
@@ -626,3 +626,14 @@
       ]
    }
 ]
+{
+      "brand": "Samsung",
+      "codename": "a70q",
+      "name": "Samsung Galaxy A70",
+      "supported_versions": [
+         {
+            "maintainer_name": "DynamicPanda",
+            "support_thread": "https://t.me/Team_Bliss_Community"
+         }
+      ]
+   },
```

Comment:
The syntax is wrong, please move this into the JSON Array, with a comma before this block and not after